pom.xml

```diff
@@ -1262,21 +1262,6 @@
             <locales>en</locales>
             <inputEncoding>${project.build.sourceEncoding}</inputEncoding>
             <outputEncoding>${project.reporting.outputencoding}</outputEncoding>
-            <!--
-              IntelliJ can't find the asciidoc config option in the site plugin, which is correct.
-              However this config section is used by the asciidoctor site plugin extension. So plead
-              ignore this error, it's actually ok.
-            -->
-            <asciidoc>
-              <attributes>
-                <source-highlighter>prettify</source-highlighter>
-                <imagesoutdir>${project.build.directory}/site/images</imagesoutdir>
-                <pom-adoc>${project.build.directory}/pom.adoc</pom-adoc>
-              </attributes>
-              <requires>
-                <require>asciidoctor-diagram</require>
-              </requires>
-            </asciidoc>
```

Comment:
Why did you remove this?

Reply:
Just wanted to make sure the reviewers aren't asleep.

(No idea and it wasn't on purpose -__- )

Reply:
The thing is, if you don't add it back again, I will not merge it ... so please update your PR and then I can merge it.